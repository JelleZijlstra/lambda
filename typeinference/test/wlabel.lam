let f = \x. x.bar in
let f' = \x. x.bar + 1 in
(print (f' {bar = 3});
print (f' {bar = 3, foo = 4});
print (f' {foo = 25});
(f, f'))
