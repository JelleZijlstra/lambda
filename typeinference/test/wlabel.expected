%%stdout%%
Error: Cannot unify types: (typevar3 with bar : int) and {foo : int}
%%stderr%%
